### PR TITLE
Add "try the beta" banner to old websites

### PR DESCRIPTION
**Pull request by didoo**
### :pushpin: Summary

This PR adds a "try the beta" banner to the old websites ("scrappy" and "flight icons").

It's ready to be merged, when it's the right time.

Preview: 
- scrappy website: https://hds-components-git-old-websites-beta-banner-hashicorp.vercel.app/
- flight website: https://hds-flight-website-git-old-websites-beta-banner-hashicorp.vercel.app/

### :link: External links

Jira ticket: https://hashicorp.atlassian.net/browse/HDS-1343

***

### 👀 Reviewer's checklist:

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-1343]: https://hashicorp.atlassian.net/browse/HDS-1343?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/components/tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -5,6 +5,9 @@
     HashiCorp Design System (HDS)
   </h1>
   <DummyNpmVersion />
+  <div class="dummy-beta-banner">
+    <strong>Notice</strong>: there is a new "beta" version of this website:
+    <a href="https://hds-website-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">do you want to try it?</a></div>
```
Comment (Dhaulagiri) [inline packages/components/tests/dummy/app/templates/application.hbs:10]:
```suggestion
    <a href="https://hds-website-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">do you want to try it?</a> Share your feedback in <a href="https://hashicorp.slack.com/archives/C04F7TH6A8P "target="_blank" rel="noopener noreferrer">#proj-ds-launch-feedback</a>.</div>
```

File: flight-website/app/templates/application.hbs [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {{page-title "Flight icons"}}
 <DsHeader />
+<div class="ds-beta-banner">
+  <strong>Notice</strong>: there is a new "beta" version of this website:
+  <a href="https://hds-website-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">do you want to try it?</a></div>
```
Comment (Dhaulagiri) [inline flight-website/app/templates/application.hbs:5]:
```suggestion
  <a href="https://hds-website-hashicorp.vercel.app/icons/library" target="_blank" rel="noopener noreferrer">do you want to try it?</a></div>
```
We _could_ link directly to the icons section if we wanted

Comment (jorytindall) [inline flight-website/app/templates/application.hbs:5]: Screen Shot 2023-01-04 at 2 18 23 PM
This link doesn't have an underline, not immediately clear that its a link. Must be overridden somewhere else

**Review by jorytindall — COMMENTED** [review]
Only one small comment (probably a CSS bug) 🚀

File: packages/components/tests/dummy/app/templates/application.hbs (hunk 1 above)
Comment (didoo) [reply packages/components/tests/dummy/app/templates/application.hbs]: 👍

File: flight-website/app/templates/application.hbs (hunk 2 above)
Comment (didoo) [reply flight-website/app/templates/application.hbs]: 👍

Comment (didoo) [reply flight-website/app/templates/application.hbs]: Fixed. Not sure how I missed it 🤦‍♂️

File: flight-website/app/styles/app.css [hunk 3]
```diff
@@ -76,6 +76,24 @@ main.ds-main {
   font-bold;
 }
 
+.ds-beta-banner {
+  display: block;
```
Comment (KristinLBradley) [inline flight-website/app/styles/app.css:80]: Question: div tags have display: block by default. Is it necessary to include this style?

File: flight-website/app/styles/app.css [hunk 4]
```diff
@@ -76,6 +76,24 @@ main.ds-main {
...
+  font-size: 1rem;
+  line-height: 1.2;
+  background-color: #f9f2ff;
+  border-radius: 0.25rem;
+}
+
+.ds-beta-banner a {
+  color: inherit;
+  text-decoration: underline;
+}
```
Comment (KristinLBradley) [inline flight-website/app/styles/app.css:95]: Comment: I would nest the styles for anchor tags inside the parent selector but don't know if you prefer not to.

File: flight-website/app/templates/application.hbs [hunk 5]
```diff
@@ -1,5 +1,14 @@
 {{page-title "Flight icons"}}
 <DsHeader />
+<div class="ds-beta-banner">
+  <strong>Notice</strong>: there is a new "beta" version of this website:
```
Comment (KristinLBradley) [inline flight-website/app/templates/application.hbs:4]: I would normally make a colon following bolded text bold also. This seems to be a common pattern but not sure if we have an accepted style rule we follow.

**Review by KristinLBradley — APPROVED** [review]
Added just a few minor comments/questions

File: flight-website/app/styles/app.css (hunk 4 above)
Comment (didoo) [reply flight-website/app/styles/app.css:95]: This is a CSS file, not a Sass one (I was confused too, at the beginning 🙂)

File: flight-website/app/styles/app.css (hunk 3 above)
Comment (didoo) [reply flight-website/app/styles/app.css]: True. Fixed now.

File: flight-website/app/templates/application.hbs (hunk 5 above)
Comment (didoo) [reply flight-website/app/templates/application.hbs]: Fixed.

**Pull request merged**